client/src/routes/contribuer/index@blank.svelte

```diff
@@ -1,27 +1,45 @@
 <script context="module" lang="ts">
+  import type { Load } from "@sveltejs/kit";
   export const prerender = true;
+  import { getTags } from "src/lib/repositories/tags";
+  import { apiToken } from "$lib/stores/auth";
+  import { get } from "svelte/store";
+  export const load: Load = async ({ fetch }) => {
+    const tags = await getTags({
+      fetch,
+      apiToken: get(apiToken),
+    });
+    return {
+      props: {
+        tags,
+      },
+    };
+  };
 </script>
 
 <script lang="ts">
   import { goto } from "$app/navigation";
   import type { DatasetFormData } from "src/definitions/datasets";
   import paths from "$lib/paths";
-  import { apiToken } from "$lib/stores/auth";
+
   import DatasetForm from "$lib/components/DatasetForm/DatasetForm.svelte";
   import { createDataset } from "$lib/repositories/datasets";
   import { Maybe } from "$lib/util/maybe";
   import DatasetFormLayout from "src/lib/components/DatasetFormLayout/DatasetFormLayout.svelte";
+  import type { Tag } from "src/definitions/tag";
 
   let loading = false;
 
+  export let tags: Tag[];
```

Comment:
Que faire si la requête de récupération des tags échoue ? Pour l'instant ce cas (certes un peu formel) n'est pas géré...

Soit on déclare un `Maybe` ici :

```suggestion
  export let tags: Maybe<Tag[]>;
```

Et dans ce cas on ajoute un `if Maybe.Some(tags)` autour du `<DatasetForm {tags} ...>`, comme tu as fais dans `edit@blank.svelte`.

Soit dans le `Load` on utilise `[]` comme valeur par défaut ? Avec une nouvelle opération de ce style : 

```ts
return {
  props: {
    tags: Maybe.withDefault([], tags)
  },
};
```

(cf https://github.com/kylecorbelli/typescript-nullable/blob/8ad79f9d32e2ad4592f0cb5802ad002bf125d2f5/src/index.ts#L37-L40)